browser/main/lib/RcParser.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash'
+
+const path = require('path')
+const sander = require('sander')
+
+function parse (boostnotercPath) {
```

Comment:
I thought they're possible:

```
$HOME/.${appname}rc
$HOME/.${appname}/config
$HOME/.config/${appname}
$HOME/.config/${appname}/config
/etc/${appname}rc
/etc/${appname}/config
```
refs: https://github.com/dominictarr/rc#standards

But if so, the name `boostnotercPath` is something wrong.